sparsity/test/test_sparse_frame.py

```diff
@@ -705,12 +705,10 @@ def test_repr():
     assert isinstance(res, str)
     assert '10x10000' in res
     assert '0 stored' in res
-    assert len(res.splitlines()) == 1 + 5 + 2
```

Comment:
Why did the lines number change? I can't see any change that you made that should affect it...

Reply:
Well, I hope our `__repr__` still makes sense then...

Reply:
It should just lines change I think depending on your terminal size? Don't know...